.github/workflows/build_test_deploy.yml

```diff
@@ -908,6 +908,46 @@ jobs:
       - run: ./scripts/publish-native.js $GITHUB_REF
       - run: ./scripts/publish-release.sh
 
+  testDeployE2E:
+    name: E2E (deploy)
+    runs-on: ubuntu-latest
+    needs: [publishRelease]
+    env:
+      NEXT_TELEMETRY_DISABLED: 1
+      NEXT_TEST_JOB: 1
+      VERCEL_TEST_TOKEN: ${{ secrets.VERCEL_TEST_TOKEN }}
+      VERCEL_TEST_TEAM: 'vtest314-next-e2e-tests'
+    steps:
+      - name: Setup node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          check-latest: true
+
+      - uses: actions/cache@v3
+        id: restore-build
+        with:
+          path: ./*
+          key: ${{ github.sha }}-${{ github.run_number }}
+
+      - run: npm i -g playwright-chromium@1.14.1 && npx playwright install-deps
```

Comment:
Should this step have a name too?

Reply:
Added in https://github.com/vercel/next.js/pull/37019/commits/dce8cb55d2c05bd8b3fade57afc3811c278e37c2